examples/addons/contentview-priority.py

```diff
@@ -0,0 +1,9 @@
+class CustomPriority(Contentview):
+    def render_priority(self, data: bytes, metadata: Metadata) -> float:
+        match metadata.content_type:
+            case "application/json":
+                return 1.0   # preferred contentview for JSON.
+            case "image/png":
+                return -1.0  # cannot handle images at all.
+            case _:
+                return 0.0
```

Comment:
Not sure I understand why the examples are not combined as before? What purpose does the `CustomPriority` example view serve if it doesn't actually implement `prettify`? Isn't that a requirement of `@abstractmethod`?

Reply:
Thinking about it... yeah. I did this so that I can have text in the middle in the docs, but we should rather fix the docs to allow partial regurgitation of examples.